### ISSUE TITLE
[ACCESSIBILITY] For IOS dropdown button items contains empty button

### ISSUE THREAD

**Issue by TodorBachvarov** [rokwire/illinois-app#4996]
For IOS only the dropdown items got two accessibility buttons: One is the actual button and the second one is the pronounced as "Button" (with no label). This issue will require additional research as I was not able to find a quick fix.
